Update mqtt-hassio.cfg

Danit2 opened the pull request:
This solves the Problem "Home Assistant Update 2024.2.0. Detected entities with a name starting with the device name"

Solves the issues https://github.com/john30/ebusd/issues/975

Danit2 committed:
Update mqtt-hassio.cfg

This solves the Problem "Home Assistant Update 2024.2.0. Detected entities with a name starting with the device name"

Danit2 committed:
Update mqtt-hassio.cfg

This alows to read and write the energy integral from vaillant.

Danit2 commented:
The second change is to alow to write the energy integral from vaillant.
It also change the read from the energyintegal to a messurement.
at the moment the energyintegral is detectet as `energy,total_increasing` and this is not valid for homeassistant.

john30 commented:
don't mix two completely independant things in one PR.
I'd merge the energy integral, but not the adaption for #975 until I had the time to look at that in depth

Danit2 committed:
Update mqtt-hassio.cfg

Delete the last change.

Danit2 committed:
Update mqtt-hassio.cfg

Alow to write the eneryintegral and change the energyintegral to a messurement.

Danit2 committed:
Merge pull request #1 from Danit2/Danit2-patch-1

Update mqtt-hassio.cfg

Danit2 committed:
Update mqtt-hassio.cfg

Danit2 committed:
Update mqtt-hassio.cfg

Danit2 commented:
Sorry for the mess.
